src/main/scala/passera/unsigned/package.scala

```diff
@@ -1,9 +1,34 @@
+/*
```

Comment:
The only change to this file was adding the banner, Removing commented code (which has been superceded), and whitespace changes.